app/views/courses/settings.html.erb

```diff
@@ -451,7 +451,7 @@
               <%= t 'course_open_enrollment', <<-TEXT, :url => enroll_url(@context.self_enrollment_code || '{{ self_enrollment_code }}'), :url2 => register_url, :code => @context.self_enrollment_code || '{{ self_enrollment_code }}', :wrapper => '<b>\1</b>'
                     This course has enabled open enrollment. Students can
                     self-enroll in the course once you share with them this URL:
-                    *%{url}*. Alternatively, they can sign up at *%{url2}* and
+                    *%{url}* . Alternatively, they can sign up at *%{url2}* and
```

Comment:
@bsnyder9 I would just change this to read as `them this URL: *%{url}* or they can sign up at` and leave the punctuation issue out of it

Reply:
Makes sense to me!